streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/KTableKTableJoinNode.java

```diff
@@ -18,12 +18,14 @@
 package org.apache.kafka.streams.kstream.internals.graph;
 
 import org.apache.kafka.streams.kstream.ValueJoiner;
+import org.apache.kafka.streams.kstream.internals.KeyValueStoreMaterializer;
```

Comment:
Meta comment: seems we have migrated the node classes in a previous PR so could you update the description of this PR to remove the statement that we changed the package here?

Reply:
ack updated